File: baekjoon_all/02000+/boj_2588.py
# Solve 2022-05-15
# Update 2023-02-22

a = int(input())
b = int(input())
print(a * (b % 10))
print(a * (b // 10 % 10))
print(a * (b // 100))
print(a * b)

File: baekjoon_all/02000+/boj_2588.cpp
// Solve 2021-12-18
// Update 2023-02-26

#include <iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;

    cout << a * (b % 10) << '\n';
    cout << a * (b / 10 % 10) << '\n';
    cout << a * (b / 100) << '\n';
    cout << a * b << '\n';

    return 0;
}